openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -276,17 +283,17 @@ Figure: Issuance using Pre-Authorized Code Flow
 
 (1) The Credential Issuer successfully obtains consent and End-User data required for the issuance of a requested Credential from the End-User using Issuer-specific business process.
 
-(2) The flow defined in this specification begins as the Credential Issuer generates a Credential Offer for certain Credential(s) and communicates it to the Wallet, for example, as a QR code or as a link.
+(2) The flow defined in this specification begins as the Credential Issuer generates a Credential Offer for certain Credential(s) and communicates it to the Wallet, for example, as a QR code or as a link. The Credential Offer contains the Credential Issuer's URL, the credential type(s) and the pre-authorized code.
 
-(3) The Wallet uses information from the Credential Offer to obtain the Credential Issuer's metadata including details about the Credential that this Credential Issuer wants to issue. This step is defined in (#credential-issuer-metadata).
+(3) The Wallet uses the Credential Issuer's URL to fetch its metadata as described in (#credential-issuer-metadata) to obtain information on the offered credential types and formats and to determine the Authorization (OAuth 2.0 Authorization Server) and Credential Endpoints.
 
-(4) The Wallet sends the Pre-Authorized Code obtained in step (2) in the Token Request to the Token Endpoint. The Wallet will send a PIN provided by the User, if it was required by the Credential Issuer. This step is defined in (#token_endpoint).  
+(4) The Wallet sends the Pre-Authorized Code obtained in step (2) in the Token Request to the Token Endpoint. The Wallet will send a User PIN provided by the User, if it was required by the Credential Issuer. This step is defined in (#token_endpoint).
```

Comment:
```suggestion
(4) The Wallet sends the Pre-Authorized Code obtained in step (2) in the Token Request to the Token Endpoint. When a User PIN is required by the Credential Issuer, the Wallet MUST send that User PIN provided by the End-User. This step is defined in (#token_endpoint).
```

sometimes we have ` Pre-Authorized Code` othertime we use `pre-authorized code`. We should align this term according the defined one

Reply:
I think Pre-Authorized Code is correct

Reply:
I'm ok with suggestions, leaving this open until we decided to capitalize verbs in this section

Reply:
Ok, probably we should open an issue to require an editorial revision of this term in the entire document and resolve this comment for now

Reply:
user PIN should be replaced with `transaction token` based on another PR.
and yes, it's `Pre-Authorized Code`

Reply:
I will keep User-PIN here  for consistency and fix it within a merge request afterwards, depending on which PR is merged first